Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM python:3.6-alpine
```

Comment:
should we create a docker-compose for the production Dockerfile as well?

Reply:
Good idea. Inspired by the [standard cookiecutter Django template](https://github.com/pydanny/cookiecutter-django/tree/master/%7B%7Bcookiecutter.project_slug%7D%7D) I think this is the threshold where we split them into separate files.